doc: use of OpenCollective and GitHub sponsors (#1658)
nodejs/TSC: Nodejs-Open-Collective.md

# Node.js Open Collective

The project maintains an Open Collective that is used to accept donations.

The funds are to be used for specific efforts as decided by the TSC and
documented on this page.

## Account details

The account is manged by the TSC. The user id and password
are shared with TSC members through 1password.

## Disbursements - management

Potential disbursements will be agreed through TSC discussion
and before approval, documented in an PR in the

> [TSC repository](https://github.com/nodejs/TSC) which adds

Reviewer: Should it be the admin repo?

Reviewer: I don’t know how consistent we are with this, my understanding is that nodejs/TSC would be for stuff internal to the TSC, so folks who are not TSC members can safely unsubscribe from that repo without missing any info relevant to them.

> the intended use and funding envelope to this page. The disbursement
> is considered approved once TSC consensus in the PR has
> been reached.

Reviewer: wdyt about adding a requirement for taking a vote if there's an objection? The idea would be to make it harder for TSC members to take controversial money decisions.
```suggestion
the intended use and funding envelope to this page. The disbursement
is considered approved if there's no objections from a project member
in 72 hours, and at least one approval from a TSC voting member.
Objections can be overridden with a TSC vote.
```

Author: @aduh95 I assume you mean an objection from a non-TSC member as a vote would already be required if there was an objection from a TSC member.

I'd be ok with that.

Reviewer 2: I'm ok with what is proposed, but I would propose something different. 

> Disbursements are approved in TSC meetings with quorum. In case of any objections, a TSC vote would be required.

Author: @mcollina I think using quorum in a meeting does not align with making sure everybody has an opportunity to comment or object. I am happy to add the the clause about objections, but I think we should clarify in terms of "objections by who". Options could include:

* TSC members
* An members of the Node.js organization
* core collaborators (although I think this is not broad enough)
* Anybody (I presonally think this is too broad)

@mcollina, @aduh95 any other options than those and thoughts on which we should specify?

# Node.js Open Collective

The project maintains an Open Collective that is used to accept donations.

The funds are to be used for specific efforts as decided by the TSC and
documented on this page.

## Account details

The account is manged by the TSC. The user id and password
are shared with TSC members through 1password.

## Disbursements - management

Potential disbursements will be agreed through TSC discussion
and before approval, documented in an PR in the
[TSC repository](https://github.com/nodejs/TSC) which adds
the intended use and funding envelope to this page. The disbursement

> is considered approved if there's no objections from a project member
> in 72 hours, and at least one approval from a TSC voting member.
> Objections can be overridden with a TSC vote.

Author: ```suggestion
is considered approved if there's no objections from a project member
in 72 hours, and consensus has been reached in the TSC. Objections can be
overridden with a TSC vote.
```

Author: @aduh95 I merged your suggetion, but have added one as I think we need TSC consensus before funds are allocated. In my opinion having one TSC approval is not enough. We need TSC consensus which is a higher bar and will take a few weeks which I think is important in this case.

Reviewer: SGTM
